core/templates/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -52,6 +53,11 @@ angular.module('oppia').directive('ckEditor4Rte', [
         var canReferToSkills = ContextService.canEntityReferToSkills();
 
         _RICH_TEXT_COMPONENTS.forEach(function(componentDefn) {
+          if (componentDefn.id === 'svgeditor') {
+            if (!ENABLE_LITERALLYCANVAS_EDITOR) {
+              return;
+            }
+          }
```

Comment:
Add TODO here for the removal of this after everything is fully implemented.

Reply:
Done.